docs/src/pages/Conformance.md

```diff
@@ -454,6 +454,14 @@ Positional Search uses [UCUM units](https://unitsofmeasure.org/ucum.html) of dis
 
 Note, the use of the surrounding bracket, such as `[mi_us]` is optional; `mi_us` is also valid.
 
+## Batch/transaction support
+The IBM FHIR Server implements the HL7 FHIR [batch/transaction](https://www.hl7.org/fhir/http.html#transaction) endpoint. We support the batch/transaction processing rules as defined at https://www.hl7.org/fhir/http.html#transaction and we support resolving and replacing both [bundle-local](https://www.hl7.org/fhir/bundle.html#references) and conditional literal references for creates and updates in a transaction bundle.
```

Comment:
Perhaps the (https://www.hl7.org/fhir/bundle.html#references) should be on the word references, instead of bundle-local.  Otherwise, LGTM.

Reply:
my original plan was to link "conditional" to the subheading under https://www.hl7.org/fhir/http.html#transaction but unfortunatley there's no `<a>` tag for that...

Reply:
i guess i could at least link it to the "trules" section:
```suggestion
The IBM FHIR Server implements the HL7 FHIR [batch/transaction](https://www.hl7.org/fhir/http.html#transaction) endpoint. We support the batch/transaction processing rules as defined at https://www.hl7.org/fhir/http.html#transaction and we support resolving and replacing both [bundle-local](https://www.hl7.org/fhir/bundle.html#references) and [conditional](https://www.hl7.org/fhir/http.html#trules) literal references for creates and updates in a transaction bundle.
```